### ISSUE TITLE
feat: sitemap

### ISSUE THREAD

**Issue by KnorpelSenf** [denoland/fresh#1305]
Is it possible to have a sitemap for sites built with fresh? If there's no build step, how would it be generated? It surely cannot work well to recurse all files on the fly in other to generate a sitemap for each request?

**Comment by marvinhagemeister** [comment]
Yes that's possible. In the file that starts the server, usually `start.ts`, you can code some logic to get the routes based on the structure of the `routes` directory. This only needs to be done once when the server starts. No need to do that on every request.

I'm hoping that we'll add a built-in way to do that sometime in the future.

**Comment by siygle** [comment]
Sorry for asking. Just want to know is there any plan for this feature, or is there any alternative solution for now?

**Comment by user451421541757324** [comment]
Any news?

**Comment by KnorpelSenf** [comment]
Yes. There is a build step now.

That makes it very easy to run e.g. my sitemap tool https://jsr.io/@knorpelsenf/sitemap before you deploy.

Nothing built-in yet AFAIK.
